src/backends/compose/compose_backend.py

```diff
@@ -278,7 +286,6 @@ def _write_docker_compose(self, warnet):
             "networks": {
                 warnet.network_name: {
                     "name": warnet.network_name,
-                    "ipam": {"config": [{"subnet": warnet.subnet}]},
```

Comment:
in "backend: remove ipv4 from compose backend" (https://github.com/bitcoin-dev-project/warnet/commit/50107b7f1dd88a1f0d393ed0e64b2ef8b19fddc7):

I'm not sure about this.. IIRC, with out patch bitcoin core will see non-routable addresses as routable (great!). But if the addresses are still in too small of a subnet, the bucketing logic wont kick in. For a compose backend, I think we might still want to set an ipam network so we can say ip addresses should be assigned in the `172.*` ranges, right?

On the other hand, if we are going to fully handle that with ASMAP (for docker and k8s backends), then yes we should nuke the network part.

Reply:
Damn you are right about the add bucketing... That is a pain. Even if I rework the asmap stuff the only way to have it loaded and effective is to know the ip addresses beforehand

Reply:
coupla ideas:

1. leave the addrman bucketing alone for now until we sort out some of the other stuff and at least get the k8s backend completely built out
2. leave this ipam network bit in for now so we can use it with the docker backend. this means addrman will work for docker with patch + defining a subnet
3. get the ASMAP stuff working where it runs after the ip addresses have been assigned (same for docker and k8s)

prefer 1) but technically thats a regression for the docker backend. 2 seems like not too much work, and 3 is gross as I feel we'll have too many moving parts. WDYT?